src/components/vuestic-components/va-slider/VaSlider.vue

```diff
@@ -744,6 +744,12 @@ export default {
       position: absolute;
       border-radius: 0.25rem;
       transition: none;
+      opacity: 0.2;
+
+      &--active,
```

Comment:
I think it would make sense to change naming a bit here.

We have
&__track - which is transparent one
&__track--inactive
&__track--active

From naming it's not apparent we have 3 states.

Instead we can use

&__track
&__track--active
&__track--background

That way
&__track &__track--active - active
&__track - inactive
&__track &__track--background - background